Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -259,15 +260,17 @@ public T GetFromMemoryCache(Uri uri)
         /// Cache specific hooks to process items from HTTP response
         /// </summary>
         /// <param name="stream">input stream</param>
+        /// <param name="initializerKeyValues">key value pairs used when initializing instance of generic type</param>
         /// <returns>awaitable task</returns>
-        protected abstract Task<T> InitializeTypeAsync(IRandomAccessStream stream);
+        protected abstract Task<T> InitializeTypeAsync(IRandomAccessStream stream, List<KeyValuePair<string, object>> initializerKeyValues);
```

Comment:
should be set to null by default to avoid breaking changes (true for all protected methods ;))